### ISSUE TITLE
Chrome won't launch after icons are applied

### ISSUE THREAD

**Issue by msimicshauma** [bil-elmoussaoui/Hardcode-Tray#463]
### Specifications
- Hardcode-Tray version : `3.5.2.1`
- Distribution : `Ubuntu 16.04`

### Expected behavior and actual behavior
After applying Papirus Icon Pack, Chrome won't launch. Every other program I use is fine. 
Then I have to reinstall chrome, and it works with it's ugly icon. I applied again and the same problem occurred. 
I'm using: "sudo hardcode-tray --conversion-tool RSVGConvert" command. Everything seems to be successfully done after running it. 
I have found a **log** and only after chrome "section" it says: **[ERROR] 2017-11-17 14:36:09 Couldn't find a PNG file.**

Everything worked fine before updating Papirus icon pack and I'm not sure if I updated Chrome.
Is there a way I can solve this? 

EDIT: Forgot to mention it's standard Google Chrome 62.0.3202.94-1

**Comment by bil-elmoussaoui** [comment]
Duplicate of #411
